tests/environment-install.bats

```diff
@@ -65,11 +65,15 @@ setup_file() {
   skip "remote environments handled in another phase"
 }
 
-@test "i?: confirmation message" {
+@test "i?: install and uninstall confirmation messages" {
   "$FLOX_CLI" init
   run "$FLOX_CLI" install hello
   assert_success
   assert_output --partial "✅ Installed 'hello' into 'test' environment."
+
+  run "$FLOX_CLI" uninstall hello
+  assert_success
+  assert_output --partial "🗑️ Uninstalled 'hello' from 'test' environment."
 }
```

Comment:
lets make this two tests since its two different commands under test here

Reply:
To make a second test, I would copy and paste this one and change the test name...that seems pointless. Did you have something else in mind?

Reply:
i mean, yeah it would be partially the same, but we'd know from the name which one failed

```
@test "u?: uninstall confirmation messages" {
  "$FLOX_CLI" init
  "$FLOX_CLI" install hello
 
  run "$FLOX_CLI" uninstall hello
  assert_success
  assert_output --partial "🗑️ Uninstalled 'hello' from 'test' environment."
}

```

Reply:
I personally find the bats output of which assertion failed already obvious enough. But I can change it